### PR TITLE
add condition is approve bad request

### PR DESCRIPTION
**Pull request by arizalpratama**
merge to main

### CODE REVIEW

File: DriverService/Data/DriverDAL.cs [hunk 1]
```diff
@@ -57,7 +57,19 @@ public void UpdatePosition(int id, Driver obj)
         //Update Accept Order -> Communication with Order Service
         public void UpdateAcceptOrder(int id, Driver obj)
```
Comment (mystald) [inline DriverService/Data/DriverDAL.cs:58]: imo lebih baik methodnya dihapus aja kalo nggak kepake

File: DriverService/Data/DriverDAL.cs [hunk 2]
```diff
@@ -57,7 +57,19 @@ public void UpdatePosition(int id, Driver obj)
...
+            // {
+            //     var result = _db.Drivers.FirstOrDefault(p => p.Id == id);
+
+            //     if (obj.IsApprove == false)
+            //     Console.WriteLine(" Status driver belum di approve ");
+            // }
+            // catch (System.Exception)
+            // {
+            //     throw;
+            // }
```
Comment (mystald) [inline DriverService/Data/DriverDAL.cs]: also, try to avoid committing lines of commented code

Comment (mystald) [reply DriverService/Data/DriverDAL.cs]: https://kentcdodds.com/blog/please-dont-commit-commented-out-code

Comment (arizalpratama) [reply DriverService/Data/DriverDAL.cs]: oke done mas

**Review by mystald — APPROVED** [review]
LGTM

**Pull request merged**